### PR TITLE
Add test coverage for RPCs containing fuelRange parameter

### PR DESCRIPTION
**Pull request by LitvinenkoIra**
`"GitHub issue link" ~ "https://github.com/SmartDeviceLink/sdl_core/issues/1656"`

### CODE REVIEW

**Comment by LitvinenkoIra** [comment]
@GetmanetsIrina please review

File: test_scripts/API/VehicleData/GetVehicleData/002_RPC_parameter_DISALLOWED_by_policies_flow.lua [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+--
+-- Requirement summary:
+-- [GetVehicleData] As a mobile app wants to send a request to get the details of the vehicle data
+--
+-- Description:
+-- In case:
+-- 1) mobile application sends valid GetVehicleData to SDL and this request is allowed
+--    by Policies but RPC parameter is not allowed
+-- SDL must:
+-- SDL responds DISALLOWED, success:false to mobile application
```
Comment (dboltovskyi) [inline test_scripts/API/VehicleData/GetVehicleData/002_RPC_parameter_DISALLOWED_by_policies_flow.lua]: @LitvinenkoIra Please remove 'SDL' word

File: test_scripts/API/VehicleData/GetVehicleData/002_RPC_parameter_DISALLOWED_by_policies_flow.lua [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+-- and doesn't transfer this request to HMI
```
Comment (dboltovskyi) [inline test_scripts/API/VehicleData/GetVehicleData/002_RPC_parameter_DISALLOWED_by_policies_flow.lua]: @LitvinenkoIra Please add list item numbers, e.g.:
```
SDL must:
1) ...
2) ...
```

File: test_scripts/API/VehicleData/GetVehicleData/001_Success_flow.lua [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  }
+}
+
+--[[ Local Functions ]]
+local function processRPCSuccess(self)
+  local mobileSession = common.getMobileSession(self, 1)
+  local cid = mobileSession:SendRPC(rpc.name, rpc.params)
+  EXPECT_HMICALL("VehicleInfo." .. rpc.name, rpc.params)
+  :Do(function(_, data)
+      self.hmiConnection:SendResponse(data.id, data.method, "SUCCESS", { fuelRange = {{type = "DIESEL", range = 45.5}} })
```
Comment (dboltovskyi) [inline test_scripts/API/VehicleData/GetVehicleData/001_Success_flow.lua]: @LitvinenkoIra Please split long lines into a few ones

File: test_scripts/API/VehicleData/commonVehicleData.lua [hunk 4]
```diff
@@ -0,0 +1,279 @@
...
+          self["mobileSession" .. id]:ExpectNotification("OnHMIStatus",
+            { hmiLevel = "NONE", audioStreamingState = "NOT_AUDIBLE", systemContext = "MAIN" })
+          :Times(1)
+          self["mobileSession" .. id]:ExpectNotification("OnPermissionsChange"):Times(2)
+          EXPECT_HMICALL("VehicleInfo.GetVehicleData", { odometer = true})
+        end)
+    end)
+end
+
+function commonVehicleData.raiN(id, self)
```
Comment (dboltovskyi) [inline test_scripts/API/VehicleData/commonVehicleData.lua]: @LitvinenkoIra Function is not used. Please remove

File: test_scripts/API/VehicleData/commonVehicleData.lua [hunk 5]
```diff
@@ -0,0 +1,279 @@
...
+  if not pAppId then pAppId = 1 end
+  return self["mobileSession" .. pAppId]
+end
+
+function commonVehicleData.getMobileAppId(pAppId)
+  if not pAppId then pAppId = 1 end
+  return config["application" .. pAppId].registerAppInterfaceParams.appID
+end
+
+function commonVehicleData.getPathToSDL()
```
Comment (dboltovskyi) [inline test_scripts/API/VehicleData/commonVehicleData.lua]: @LitvinenkoIra Function is not used. Please remove

**Comment by LitvinenkoIra** [comment]
@dboltovskyi Fixed all comments c2671d39aabe339c281200f827b71f75d4e89c61

**Pull request merged**